### PR TITLE
Just some changes on dev environment

### PR DESCRIPTION
**Pull request by schleumer**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -41,3 +41,7 @@ config :code_stats, CodeStats.Repo,
   database: "code_stats_dev",
   hostname: "localhost",
   pool_size: 10
+
+# To avoid conflicts with workspaces
+# dev.secret.exs is not versioned, like prod.secret.exs
+import_config "dev.secret.exs"
```
Comment (Nicd) [inline config/dev.exs:47]: 👍

File: package.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
 {
   "license": "Unlicensed",
   "private": true,
-  "repository": {
-  },
+  "repository": {},
   "dependencies": {
     "babel-brunch": "~6.0.0",
+    "babel-preset-es2015": "^6.9.0",
```
Comment (Nicd) [inline package.json:7]: What do we need this for? Or just planning for the future?

Comment (schleumer) [inline package.json:7]: It's used on Babel Brunch for transpile Javascript files, when it's not present Phoenix throws an exception.

```
19 Jul 01:39:44 - error: Compiling of web\static\js\app.js failed. Couldn't find preset ".\\node_modules\\babel-preset-es2015" relative to directory "web\\static\\js" ; Compiling of web\static\js\socket.js failed. Couldn't find preset ".\\node_modules\\babel-preset-es2015" relative to directory "web\\static\\js"
```

Comment (Nicd) [inline package.json:7]: Would have thought that would be added into the package.json by default then. :o  Nice catch. I haven't used any JavaScript in the project so I didn't notice that.

**Pull request merged**